public/js/scripts.js

```diff
@@ -30,13 +30,13 @@ window.addEventListener('DOMContentLoaded', event => {
     document.addEventListener('scroll', navbarShrink);
 
     // Activate Bootstrap scrollspy on the main nav element
-    const mainNav = document.body.querySelector('#mainNav');
-    if (mainNav) {
-        new bootstrap.ScrollSpy(document.body, {
-            target: '#mainNav',
-            offset: 72,
-        });
-    };
+    // const mainNav = document.body.querySelector('#mainNav');
+    // if (mainNav) {
+    //     new bootstrap.ScrollSpy(document.body, {
+    //         target: '#mainNav',
+    //         offset: 72,
+    //     });
+    // };
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/GN4RK/p5-blog/pullRequest?prid=9579791)
```suggestion
  // };
```